dist/docker-images/ziti-router/compose.yml

```diff
@@ -6,18 +6,17 @@ volumes:
 services:
   chown-router:
     image: busybox
-    command: chown -R ${ZIGGY_UID:-2171} /mnt
+    command: chown -R ${ZIGGY_UID:-2171} /ziti-router
```

Comment:
same as controller, use a working dir named after the image to avoid collisions with commonly-used, generic mountpoints